### PR TITLE
[docs] Fixed info at apiserver.certSANs spec

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Fixed info at apiserver.certSANs spec.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Fixed info at apiserver.certSANs spec.
impact_level:  low
```

### CODE REVIEW

File: modules/040-control-plane-manager/openapi/config-values.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ properties:
           * `kubernetes.default`;
           * `kubernetes.default.svc`;
           * `kubernetes.default.svc.cluster.local`;
-          * `192.168.0.1`;
+          * *kubernetes API address*;
```
Comment (mortelumina98) [inline modules/040-control-plane-manager/openapi/config-values.yaml:108]:
```suggestion
          * *Kubernetes API address*;
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ properties:
           * `kubernetes.default`;
           * `kubernetes.default.svc`;
           * `kubernetes.default.svc.cluster.local`;
-          * `192.168.0.1`;
+          * *адрес kubernetes API*;
```
Comment (mortelumina98) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:62]:
```suggestion
          * *адрес Kubernetes API*;
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -59,7 +59,7 @@ properties:
           * `kubernetes.default`;
           * `kubernetes.default.svc`;
           * `kubernetes.default.svc.cluster.local`;
-          * `192.168.0.1`;
+          * *адрес Kubernetes API*;
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:62]:
```suggestion
          * Адрес API-сервера кластера;
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -59,7 +59,7 @@ properties:
...
           * `127.0.0.1`;
           * *текущий_hostname*;
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:64]:
```suggestion
          * DNS-имя хоста (hostname);
```

File: modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml [hunk 5]
```diff
@@ -59,7 +59,7 @@ properties:
...
           * *hostIP*.
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml:65]:
```suggestion
          * IP-адрес хоста.
```

File: modules/040-control-plane-manager/openapi/config-values.yaml [hunk 6]
```diff
@@ -105,7 +105,7 @@ properties:
           * `kubernetes.default`;
           * `kubernetes.default.svc`;
           * `kubernetes.default.svc.cluster.local`;
-          * `192.168.0.1`;
+          * *Kubernetes API address*;
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/config-values.yaml:108]:
```suggestion
          * Kubernetes API server address;
```

File: modules/040-control-plane-manager/openapi/config-values.yaml [hunk 7]
```diff
@@ -105,7 +105,7 @@ properties:
...
           * `127.0.0.1`;
           * *current_hostname*;
           * *hostIP*.
```
Comment (z9r5) [inline modules/040-control-plane-manager/openapi/config-values.yaml:111]:
```suggestion
          * host name;
          * host IP.
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10186](https://github.com/deckhouse/deckhouse/pull/10186) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**